rules/cross-platform/privilege_escalation_sudo_buffer_overflow.toml

```diff
@@ -0,0 +1,87 @@
+[metadata]
+creation_date = "2021/02/03"
+maturity = "production"
+updated_date = "2021/02/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies the attempted use of a Heap-based Buffer Overflow vulnerability for the Sudo binary in Unix-like
+systems (CVE-2021-3156). Successful exploitation allows an unprivileged user to escalate to the root user.
+"""
+false_positives = [
+    """
+    This rule could generate false positives if the process arguments leveraged by the expoit are shared by custom
+    scripts using the Sudo or Sudoedit binaries.
+    """,
+]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Possible Sudo Heap-based Buffer Overflow Vulnerability Attempt (CVE-2021-3156)"
+note = "The rule identifies attempted exploitation of a privilege escalation vulnerability for Sudo. Successful exploitation could allow arbitrary code to be run on the target system with elevated privileges. Alerts from this rule should be invested to identify the validility of the process executions."
```

Comment:
The note field is really for investigation steps. If it describes the same thing as the description, its probably not needed.